bibtex-completion.el

```diff
@@ -1030,16 +1046,20 @@ line."
 (defun bibtex-completion-show-entry (keys)
   "Show the first selected entry in the BibTeX file."
   (catch 'break
-    (dolist (bibtex-file (-flatten (list bibtex-completion-bibliography)))
+    (dolist (bib-file (bibtex-completion-normalize-bibliography 'main))
       (let ((key (car keys))
-            (buf (bibtex-completion-buffer-visiting bibtex-file)))
-        (find-file bibtex-file)
-        (goto-char (point-min))
-        (if (re-search-forward
-             (concat "^@\\(" parsebib--bibtex-identifier
-                     "\\)[[:space:]]*[\(\{][[:space:]]*"
-                     (regexp-quote key) "[[:space:]]*,") nil t)
-            (throw 'break t)
+            (buf (bibtex-completion-buffer-visiting bib-file)))
+        (find-file bib-file)
+        (if (eq major-mode 'org-mode)
+            (when (ignore-errors (org-link-search (concat "#" key)))
```

Comment:
You say the the bibliography and the notes file can be the same.  Since we `org-narrow-to-subtree` when showing notes, do we have to `widen` here in order to find the entry?